resources/views/inc/navbar.blade.php

```diff
@@ -21,6 +21,7 @@
                         <a class="dropdown-item" href="/pilots/airports">Airports</a>
                         <a class="dropdown-item" href="/pilots/request-staffing">Request Staffing</a>
                         <a class="dropdown-item" href="http://www.flightaware.com/statistics/ifr-route/" target="_blank">IFR Routes</a>
+                        <a class="dropdown-item" href="/pilots/guide/atl">ATL Pilot Guide</a>
 						<a class="dropdown-item" href="/ramp-status/atl">ATL Gate Status</a>
 						<a class="dropdown-item" href="/ramp-status/clt">CLT Gate Status</a>
```

Comment:
Do you like eslint? Default in VS Code?

Reply:
Is eslint a linting tool or formatting tool? There's CLI tools that we can just run before we commit that will automatically format, and it can be configured in the repository so we don't have to worry about local config differences.

Reply:
I'm fine with whatever standard you prefer. For now, I'll use VS Code's default indentation

Reply:
Fixed indentations and moved CSS and JS to separate asset files.